processing/src/main/java/org/apache/druid/query/aggregation/LongSumAggregatorFactory.java

```diff
@@ -36,12 +36,28 @@ public class LongSumAggregatorFactory extends SimpleLongAggregatorFactory
 {
   private final Supplier<byte[]> cacheKey;
 
-  @JsonCreator
+  /**
+   * If true, the aggregator will return null when there are no values to aggregate. This is the default behavior.
+   * Otherwise, the aggregator must return a default non-null value. This is only used internally and not user-facing.
+   */
+  private final boolean isNullable;
```

Comment:
i think this should also be `forceNotNullable` like `NullableNumericAggregatorFactory`